core/build.sbt

```diff
@@ -46,8 +46,6 @@ pomExtra := {
   </developers>
 }
 
-osgiSettings
```

Comment:
Hmm, I'm not familiar either... I though this was a straightforward change to auto-plugins. I'll take a deeper look later. Thanks :)